packages/linters/src/eslint.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
     node: true,
     browser: true,
     jest: true,
+    es6: true,
```

Comment:
(question) `eslint` already has others ES environments like es6, es2017, es2020, and es2021

Does it make sense to use updated environments here?

Reply:
Excellent observation. [Change to es2021](https://github.com/juntossomosmais/time-out-market/pull/104/commits/002c3a734f15527382400de5f4c450073993c19c)

@felipefialho